src/component/Canvas.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import Node from './Node';
+import Edge from './Edge';
+
+function calulateCoordinates(
+  nodes: string[],
```

Comment:
dunno but `Array<string>` would be way more better for me, `string[]` looks too c-ish